Modules/Filtering/Smoothing/include/itkFFTDiscreteGaussianImageFilter.h

```diff
@@ -0,0 +1,141 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkFFTDiscreteGaussianImageFilter_h
+#define itkFFTDiscreteGaussianImageFilter_h
+
+#include "itkDiscreteGaussianImageFilter.h"
+
+#include "itkForwardFFTImageFilter.h"
+#include "itkImage.h"
+#include "itkInverseFFTImageFilter.h"
+#include "itkMacro.h"
+
+namespace itk
+{
+/**
+ * \class FFTDiscreteGaussianImageFilter
+ * \brief Blurs an image by convolution with a discrete gaussian kernel
+ * in the frequency domain.
+ *
+ * This filter performs Gaussian blurring by convolution of an image
+ * and a discrete Gaussian operator (kernel) in the frequency domain
+ * by way of Fast Fourier Transform forward and inverse operations.
+ *
+ * \sa GaussianOperator
+ * \sa Image
+ * \sa Neighborhood
+ * \sa NeighborhoodOperator
+ * \sa RecursiveGaussianImageFilter
+ *
+ * \ingroup ImageEnhancement
+ * \ingroup ImageFeatureExtraction
+ * \ingroup ITKSmoothing
+ *
+ * \sphinx
+ * \sphinxexample{Filtering/Smoothing/SmoothWithRecursiveGaussian,Computes the smoothing with Gaussian kernel}
```

Comment:
Does sphinx example `SmoothWithRecursiveGaussian` use `FFTDiscreteGaussianImageFilter`? If no, this line should be removed.